modules/ROOT/pages/op-tkn-cli-reference.adoc

```diff
@@ -0,0 +1,25 @@
+[id='pipeline-cli-reference']
+= Pipeline CLI reference
+//include::modules/openshift-developer-cli-attributes.adoc[]
```

Comment:
please remove this we will have our pipeline attributes

Reply:
done